### ISSUE TITLE
Change NULL to nullptr

### ISSUE THREAD

**Issue by gregoryphealy** [chang-s/astro#10]
Just to be a bit more in compliance with the C++11 standard. NULL also depends on STL libraries, so it'll reduce any risk of error associated with STLs not being there.

**Comment by strawnpe** [comment]
Good call. This can be done pretty simply with vim once we get to this stage: `%s/NULL/nullptr/g`.
